backend/Recodo.DAL/Entities/AccessForUnregisteredUsers.cs

```diff
@@ -0,0 +1,10 @@
+namespace Recodo.DAL.Entities
+{
+    public class AccessForUnregisteredUsers
```

Comment:
Do not update anything, but `GuestsAccess` would be much more concise in this case